@types/phaser3-rex-plugins/plugins/behaviors/textedit/TextEdit.d.ts

```diff
@@ -0,0 +1,29 @@
+declare module 'phaser3-rex-plugins/plugins/behaviors/textedit/TextEdit' {
+    import InputText, {TextChangedCallback} from 'phaser3-rex-plugins/plugins/gameobjects/dom/InputText';
+
+    export interface OnTextEditClosed {
```

Comment:
Maybe better to rename this type to `OnCloseCallback` and simplify like this `type OnCloseCallback = (obj: Phaser.GameObjects.GameObject) => void;`